### ISSUE TITLE
Error Message for attaching a file

### ISSUE THREAD

**Issue by BobbyBaileyRB** [department-of-veterans-affairs/vets-design-system-documentation#2281]
# Bug Report

- [Y] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

I noticed that in Secure Messaging the design(s) for errors when attaching a file does not have the left border. Reviewing the design system [file inputs](https://design.va.gov/components/form/file-input) it also does not have it. However, it is being shown in storybook.

![Screenshot 2023-11-28 at 4 52 21 PM](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/102695954/aa68add5-306f-4e5e-b2d2-de792b34e91a)

![Screenshot 2023-11-28 at 4 52 35 PM](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/102695954/9871d55d-a881-4691-bca5-403aa9e1ec51)

![SM color](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/102695954/f53ee67c-a973-4875-a8e5-2bf46ba7ebdf)

![SM color blindness](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/102695954/2abe19f7-976c-4ece-b821-b33018501737)

## What I expected to happen

Expected to have the side border when an error happens while attaching a file.

## Reproducing
- Formation version:
- Device: (Macbook)
- Browser: (Chrome)

Steps to reproduce:

No known steps to reproduce. There bug seems to be consistent. 

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

**Comment by oleksii-morgun-va** [comment]
I was chatting with other devs who were previously involved with Secure Messaging. The reason why we had to not use the Design System component is due to specific design and behavior our UCD team came up with. We had to use one button to attach multiple attachments to the same form, and there is a custom design to already attached files with it's own buttons. I don't think that the task was to match what is in the design system, but rather match the UCD mocks
